pr_stats.c

```diff
@@ -2528,7 +2528,7 @@ __print_funct_t stub_print_filesystem_stats(struct activity *a, int curr, int di
 		       sfc->f_files - sfc->f_ffree,
 		       sfc->f_files ? SP_VALUE(sfc->f_ffree, sfc->f_files, sfc->f_files)
 				    : 0.0,
-		       sfc->fs_name);
+		       DISPLAY_MOUNT(a->opt_flags) ? sfc->mountp : sfc->fs_name);
```

Comment:
pr_stats.c has been modified to display mountpoints. But there are other files to update for the various output formats used by sadf (json_stats.c, xml_stats.c, rndr_stats.c)